src/shared-components/Recurrence/CustomRecurrenceModal.tsx

```
@@ -0,0 +1,596 @@
+              <span className="fw-semibold text-secondary">
+                {t('repeatsOn')}
+              </span>
+              <br />
+              <div className="mx-2 mt-3 d-flex gap-1">
+                {daysOptions.map((day, index) => (
+                  <div
+                    key={index}
+                    className={`${styles.recurrenceDayButton} ${byDay?.includes(Days[index]) ? styles.selected : ''}`}
+                    onClick={() => handleDayClick(Days[index])}
+                    data-testid="recurrenceWeekDay"
+                  >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)